std/algebra/emulated/sw_bw6761/pairing.go

```diff
@@ -432,6 +432,56 @@ func (pr Pairing) doubleAndAddStep(p1, p2 *g2AffP) (*g2AffP, *lineEvaluation, *l
 	return &p, &line1, &line2
 }
 
+// doubleAndSubStep doubles p1 and subs p2 to the result in affine coordinates, and evaluates the line in Miller loop
+// https://eprint.iacr.org/2022/1162 (Section 6.1)
+func (pr Pairing) doubleAndSubStep(p1, p2 *g2AffP) (*g2AffP, *lineEvaluation, *lineEvaluation) {
```

Comment:
Similarly, would only have a single method `doubleAndAddStep` and then boolean argument `isSub` etc. and then conditionally either add or sub on line 443.